q2_feature_classifier/_vsearch.py

```diff
@@ -81,3 +192,57 @@ def classify_consensus_vsearch(query: DNAFASTAFormat,
                  'choosing the top N hits, not the first N hits.'),
     citations=[citations['rognes2016vsearch']]
 )
+
+
+plugin.pipelines.register_function(
+    function=classify_hybrid_vsearch_sklearn,
+    inputs={**inputs, 'classifier': TaxonomicClassifier},
+    parameters={**parameters,
+                'reads_per_batch': Int % Range(0, None),
+                'confidence': Float,
```

Comment:
```suggestion
                'confidence': Float % Range(0, 1, inclusive_start=False, inclusive_end=True) | Str % Choices(['disable', 'calculate_without_application']),
```

How does that look to you? I have no clue about the sentinel names, so edit away.

Reply:
again, trying to mirror the underlying params in this pipeline — and the underlying params need an update, so let me push up another commit with those changes.